RELEASE.md

```diff
@@ -1,20 +1,20 @@
 # Release
 
-To release new version of Kuma follow the steps:
+To release a new version of Kuma follows the steps:
```

Comment:
I think, the original `follow the steps` might be the right version.

Maybe we could change it to `follow the steps below` instead ?

Reply:
Thank You @yskopets  for review, Yeah I am changing this